extras/claude.sh

```
@@ -0,0 +1,169 @@
+done
+
+log "Waiting for all planning agents..."
+failed=0
+for i in "${!pids[@]}"; do
+    if ! wait "${pids[$i]}" 2>/dev/null; then
+        log "  Warning: agent $((i+1)) failed (PID ${pids[$i]})"
+        (( failed++ )) || true
+    fi
+done
+
+plan_count=$(ls plan.*.md 2>/dev/null | wc -l)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)